### PR TITLE
#32335: Adding initial files to .github

### PR DESCRIPTION
**Pull request by rl1987**
https://trac.torproject.org/projects/tor/ticket/32335

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+Contributing to Tor
```
Comment (teor2345) [inline CONTRIBUTING.md]: Please add a list of projects, and then put each project in its own section.

It's ok if the list just has "Tor (the network daemon)" for now.

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Reporting Tor vulnerabilities
+=============================
+
+Please report vulnerabilities and security issues to tor-security@lists.torproject.org. 
+
+If you found vulnerability in core Tor daemon or Tor Browser, consider submitting it to bug bounty program at: https://hackerone.com/torproject
```
Comment (teor2345) [inline SECURITY.md]:
```suggestion
If you have found a vulnerability in the Tor network daemon or Tor Browser, consider submitting it to bug bounty program at: https://hackerone.com/torproject
```

File: SUPPORT.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Tor Support
```
Comment (teor2345) [inline SUPPORT.md]: Please add a list of projects, and then put each project in its own section.

It's ok if the list just has "Tor (the network daemon)" for now.

File: FUNDING.yml [hunk 4]
```diff
@@ -0,0 +1 @@
+custom: "https://donate.torproject.org/"
```
Comment (ahf) [inline FUNDING.yml]: Could this URL be changed to https://torproject.org/donate/donate-github ?

**Pull request closed without merging**